proj/proj.go

```diff
@@ -215,30 +213,42 @@ func CloneGeometry(geometry geom.Geometry) (geom.Geometry, error) {
 func ToWebMercator(SRID uint64, geometry geom.Geometry) (geom.Geometry, error) {
 	switch SRID {
 	default:
-		return nil, fmt.Errorf("don't know how to convert from %v to %v.", WebMercator, SRID)
-	case WebMercator:
+		return nil, fmt.Errorf("don't know how to convert from %v to %v.", proj.WebMercator, SRID)
+	case proj.WebMercator:
 		// Instead of just returning the geometry, we are cloning it so that the user of the API can rely
 		// on the result to alway be a copy. Instead of being a reference in the on instance that it's already
 		// in the same SRID.
 
 		return CloneGeometry(geometry)
```

Comment:
@ear7h you have an [open issue](https://github.com/go-spatial/geom/issues/42) around adding `Clone()` to the geom.Geometry interface. If you implement that and port over the tegola `proj.ApplyToPoints` func in I think we can make this PR even cleaner.